Source/FolioReaderFontsMenu.swift

```diff
@@ -234,7 +234,14 @@ class FolioReaderFontsMenu: UIViewController, SMSegmentViewDelegate, UIGestureRe
         layoutDirection.tag = 3
         layoutDirection.addSegmentWithTitle(readerConfig.localizedLayoutVertical, onSelectionImage: verticalSelected, offSelectionImage: verticalNormal)
         layoutDirection.addSegmentWithTitle(readerConfig.localizedLayoutHorizontal, onSelectionImage: horizontalSelected, offSelectionImage: horizontalNormal)
-        layoutDirection.selectSegmentAtIndex(Int(FolioReader.currentScrollDirection))
+
+        var scrollDirection = FolioReaderScrollDirection(rawValue: FolioReader.currentScrollDirection)
```

Comment:
Can't remember why didn't it require me to add `?? .vertical` here too, as on the line #98.